server/server.go

```diff
@@ -613,14 +613,17 @@ func New(config *Config) (*Server, error) {
 	s.restore()
 	s.cleanupSandboxesOnShutdown()
 
-	bindAddress, err := knet.ChooseBindAddress(net.IP{0, 0, 0, 0})
-	if err != nil {
-		return nil, err
+	bindAddress := net.ParseIP(config.StreamAddress)
+	if bindAddress == nil {
+		bindAddress, err = knet.ChooseBindAddress(net.IP{0, 0, 0, 0})
+		if err != nil {
+			return nil, err
+		}
 	}
 
 	// Prepare streaming server
 	streamServerConfig := streaming.DefaultConfig
-	streamServerConfig.Addr = bindAddress.String() + ":10101"
+	streamServerConfig.Addr = net.JoinHostPort(bindAddress.String(), config.StreamPort)
```

Comment:
This value ultimately gets passed into `net.Listen`, which calls `net.LookupPort`. I'm fine either way.